spec/standard-webhooks.md

```diff
@@ -138,9 +138,9 @@ The unique identifier is a unique identifier associated with a specific event tr
 
 #### Signature scheme
 
-As mentioned above, it's important to sign both the body of the webhook, and the associated metadata. To achieve this, the metadata and the body are concatenated (delimited by full-stops) and then signed.
+As mentioned above, it's important to sign both the body of the webhook and the associated metadata. To achieve this, the message's ID, timestamp and body are concatenated - delimited by full stops - and then signed.
```

Comment:
```suggestion
As mentioned above, it's important to sign both the body of the webhook and the associated metadata. To achieve this, the message's ID, timestamp and body are concatenated—delimited by full stops—and then signed.
```
Em dashes aren't surrounded by spaces + change to em dash instead of a hyphen.

Reply:
I forced push changes; therefore, they do not appear here. To see the latest code, check https://github.com/standard-webhooks/standard-webhooks/pull/117/files